components/webui/imports/ui/IngestView/panels/IngestionJobs/IngestionJobRow.jsx

```diff
@@ -37,19 +40,32 @@ const COMPRESSION_JOB_STATUS_ICONS = Object.freeze({
  * @return {React.ReactElement}
  */
 const IngestionJobRow = ({job}) => {
-    let speedText = "";
     let uncompressedSizeText = "";
+    let speedText = "";
```

Comment:
why we reorder this?

Reply:
To match the order of those variables' assignments. We could revert it back to match the order of the columns though. What do you think?